README.md

```diff
@@ -1,17 +1,25 @@
-# djcat-visual-stimuli
-The `stimulus` schema is a self-contained application that generates and presents visual stimuli using PsychToolbox and records conditions and trials in a DataJoint database.
+# DataJoint Element - Visual Stimulus
++ The `stimulus` schema is a self-contained application that generates and presents visual stimuli using [PsychToolbox](http://psychtoolbox.org) and records conditions and trials in a DataJoint database.
```

Comment:
Their documentation looks like it only has a capitalized P. Revising to follow the same convention

```suggestion
+ The `stimulus` schema is a self-contained application that generates and presents visual stimuli using [Psychtoolbox](http://psychtoolbox.org) and records conditions and trials in a DataJoint database.
```